WildFly_30.0.1.Final/jakarta-core-jdk21.adoc

```diff
@@ -0,0 +1,57 @@
+= Jakarta Core Profile 10.0.1 TCK Java SE 17 Results
+:ext-relative: {outfilesuffix}
+:imagesdir: ../images/
+
+image:splash_wildflylogo_small.png[WildFly, align="center"]
+
+
+=== Compatibility Certification Information
+----
+Date:  October 18, 2023
+Product Name, Version: WildFly 30.0.0.Final
```

Comment:
30.0.1?

Reply:
Also the date seems a little shady, so probably a bad paste

Reply:
> 30.0.1?

Nice catch, thank you Martin!

> Also the date seems a little shady, so probably a bad paste

Nice catch, thank you Martin!